xml/obs_moderation.xml

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.1" xml:id="cha.obs.moderation"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink" >
+  <title>Moderation</title>
+  <info/>
+ 
+  <para>As a platform where a lot of social interaction happens, OBS is not free of spam,
+    harmful content or other outcomes of user misconduct.
+    Therefore, OBS provides features which help any user report problematic content and help moderators to act accordingly.
+    Those features are described below, and they are helpful for admins, staff members, moderators and users.
```

Comment:
Shouldn't those be different `<para>`?

Reply:
No, I intend to have one paragraph with three lines. That's how it looks in the resulting document.

I separated the lines in the code to make it easier to read.

Reply:
Alright